lineage/tHMM.py

```diff
@@ -106,7 +106,7 @@ def get_BIC(self, LL: float, num_cells: int, atonce=False) -> Tuple[float, float
             degrees_of_freedom += self.num_states * (self.num_states - 1)
 
         if atonce:  # assuming we are fitting all 4 concentrations at once and we have cell cycle phase specific distributions.
-            degrees_of_freedom += self.num_states * 4.5
+            degrees_of_freedom += self.num_states * 2.25
```

Comment:
Does this therefore change some of the existing figures?

Reply:
no, it was correct for the phase-specific observations (since we have two phases, it would be 2 * 2.25).

Reply:
Aren't you changing it for all situations here?

Reply:
I wrote an if statement to separate mcf10a from au565 case.